cmd/api/src/daemons/datapipe/jobs.go

```diff
@@ -214,7 +216,9 @@ func (s *Daemon) processIngestFile(ctx context.Context, path string, fileType mo
 
 				if err := file.Close(); err != nil {
 					log.Errorf("Error closing ingest file %s: %v", filePath, err)
-				} else if err := os.Remove(filePath); err != nil {
+				} else if err := os.Remove(filePath); errors.Is(err, fs.ErrNotExist) {
+					log.Warnf("Removing ingest file %s: %v", filePath, err)
```

Comment:
Nit: `%w` probably makes more sense than `%v`.
While they are synonymous, `%w` enforces the `Error` interface to be implemented, which for a lot of the warnings we have, it seems like that's the case that we want.

I see `%v` was already being used, and we use it a lot elsewhere, so moreso just an observation

Reply:
thanks for the explanation!  went ahead and updated.